presidio-anonymizer/presidio_anonymizer/entities/analyzer_result.py

```diff
@@ -78,15 +78,16 @@ def __str__(self):
         return f"start: {str(self.start)}, end: {str(self.end)}, " \
                f"score: {str(self.score)}, entity_type: {self.entity_type}"
 
-    def same_or_contained(self, other):
+    def has_conflict(self, other):
```

Comment:
"conflicting" is a little confusing IMHO. Consider explaining what it means in the docstring or change the name.

Reply:
Will explain in doc string.